web/packages/teleport/src/AppLauncher/AppLauncher.tsx

```diff
@@ -63,22 +63,14 @@ export function AppLauncher() {
 
       let path = '';
       if (queryParams.has('path')) {
-        path = decodeURIComponent(queryParams.get('path'));
```

Comment:
+1 for removing an unnecessary (as far as I understand) call to `decodeURIComponent`.